# Array of keypoint pixel locations.

Header header
Keypoint2D[] keypoints_2D
